tests/test_security_path_traversal.py

```
@@ -112,3 +112,77 @@ def test_run_inspector_validation():
+        manager.compile_prompt_plan("valid_flow", "step", run_id="../bad")
+
+
+def test_run_state_manager_path_validation(tmp_path):
+    """Test that RunStateManager validates path components against traversal."""
+    import asyncio
+
+    from swarm.api.services.run_state import RunStateManager
+
+    manager = RunStateManager(runs_root=tmp_path)
+
+    async def run_tests():
```

Ruff (0.14.13)

Missing return type annotation for private function `run_tests`

Add return type annotation: `None`

(ANN202)